plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/util/ResourceUtils.java

```diff
@@ -16,15 +16,27 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 import org.apache.commons.lang.StringUtils;
+import org.jboss.tools.openshift.core.OpenShiftAPIAnnotations;
 
+import com.openshift.restclient.ResourceKind;
 import com.openshift.restclient.capability.CapabilityVisitor;
 import com.openshift.restclient.capability.resources.ITags;
+import com.openshift.restclient.model.IBuild;
+import com.openshift.restclient.model.IBuildConfig;
+import com.openshift.restclient.model.IObjectReference;
+import com.openshift.restclient.model.IPod;
 import com.openshift.restclient.model.IResource;
+import com.openshift.restclient.model.IService;
+import com.openshift.restclient.model.deploy.IDeploymentImageChangeTrigger;
 
 public class ResourceUtils {
 
+	public static final String DOCKER_IMAGE_KIND = "DockerImage";
```

Comment:
@adietish I dont know if these are official kinds that you can 'get' and 'list' though we do have some sub resources and such on ResourceKind.